tox.ini

```diff
@@ -21,9 +21,8 @@ deps =
     test: pytest-cov
     test: pytest-timeout
 commands_pre =
-    # Build ioutil for both current python (e.g. 3.8) and platform python (e.g. 3.6)
+    # Build ioutil for current python (e.g. 3.8)
     python setup.py build_ext --build-lib .
-    /usr/bin/python3 setup.py build_ext --build-lib .
```

Comment:
This fail with an error suggesting to include this version in allow_externals, so this will work also when runing inside the virtual environment. Maybe this is a better solution?

Or maybe this is not needed now since we change the way we run the tests?

Can you run the tests locally with this change?

Reply:
Yes, I can run them locally.

The problem seemed to be that `python` and `/usr/bin/python` pointed to different Python versions (3.8 and 3.6, respectively), and we needed the `ioutil` package to be installed in both when running in the container, as it was using the platform python.

With the `venv` they all point to the same version (e.g.):
```
lrwxrwxrwx. 1 root root    7 Feb 15 08:12 python -> python3
lrwxrwxrwx. 1 root root   16 Feb 15 08:12 python3 -> /usr/bin/python3
lrwxrwxrwx. 1 root root    7 Feb 15 08:12 python3.11 -> python3
```
So we always could run this locally, problem was with only for the containers.